cinema/models.py

```diff
@@ -35,12 +39,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def create_custom_path(instance, filename: str) -> str:
+    _, extension = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{extension}"
+    return os.path.join("uploads/images/", filename)
```

Comment:
due to task requirements:
uploads/movies/

Reply:
Also i recoment to do as:
```suggestion
    return os.path.join("uploads", "movies", filename)
```